editor/plugins/script_text_editor.cpp

```diff
@@ -1779,6 +1797,7 @@ ScriptTextEditor::ScriptTextEditor() {
 	code_editor->set_code_complete_func(_code_complete_scripts, this);
 	code_editor->get_text_edit()->connect_compat("breakpoint_toggled", this, "_breakpoint_toggled");
 	code_editor->get_text_edit()->connect_compat("symbol_lookup", this, "_lookup_symbol");
+	code_editor->get_text_edit()->connect_compat("symbol_validation", this, "_validate_symbol");
```

Comment:
The signal name doesn't match its registration... so that's likely not working at all :)

Reply:
Ahh, I forget to update it. :sweat_smile: